Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -481,11 +504,7 @@ def get_incidents_batch_by_time_request(params):
         request_params['created_after'] = created_after.isoformat().split('.')[0] + 'Z'
         request_params['created_before'] = created_before.isoformat().split('.')[0] + 'Z'
 
-    # fetching the last batch
-    request_params['created_before'] = current_time.isoformat().split('.')[0] + 'Z'
-    incidents_list.extend(get_incidents_request(request_params))
-
-    return incidents_list
+    return incidents_list, last_fetch, new_fetched_incidents
```

Comment:
let's log the nunber of `incidents`, `filtered_incidents` and `incidents_list` for future bugs.